src/query-builder/delete-query-builder.ts

```diff
@@ -498,12 +498,16 @@ export class DeleteQueryBuilder<DB, TB extends keyof DB, O>
     })
   }
 
-  returningAll(): DeleteQueryBuilder<DB, TB, Selectable<DB[TB]>> {
+  returningAll(): DeleteQueryBuilder<DB, TB, Selectable<DB[TB]>>
+
+  returningAll<T extends TB>(table: T): DeleteQueryBuilder<DB, TB, DB[T]>
```

Comment:
1. Let's support multi-table, just like `SelectQueryBuilder.selectAll([t1, t2])`..

Nit

2. Let's swap the overloads so it reads from specific to non-specific just like in `SelectQueryBuilder.selectAll`.

```suggestion
  returningAll<T extends TB>(tables: ReadonlyArray<T>): DeleteQueryBuilder<DB, TB, DB[T]>

  returningAll<T extends TB>(table: T): DeleteQueryBuilder<DB, TB, DB[T]>

  returningAll(): DeleteQueryBuilder<DB, TB, Selectable<DB[TB]>>
```

Reply:
Updated to support multi-table.